release/docker/build-image-multi-arch.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+set -e
+
+# Import libs
+. ../../lib/shell/file_management.sh
+
+# Variable
+OLDIFS=$IFS
+BUILDER_NUM=`date +%s`
+BUILDER_NAME="multiarch_${BUILDER_NUM}"
+DIR=""
+
+function usage() {
+    echo ""
+    echo "This script is used to build the OpenSearch Docker image with single architecture (x64 or arm64). It prepares the files required by the Dockerfile in a temporary directory, then builds and tags the Docker image."
+    echo "--------------------------------------------------------------------------"
+    echo "Usage: $0 [args]"
+    echo ""
+    echo "Required arguments:"
+    echo -e "-v VERSION\tSpecify the OpenSearch version number that you are building, e.g. '1.0.0' or '1.0.0-beta1'. This will be used to label the Docker image. If you do not use the '-o' option then this tool will download a public OPENSEARCH release matching this version."
+    echo -e "-f DOCKERFILE\tSpecify the dockerfile full path, e.g. dockerfile/opensearch.al2.dockerfile."
+    echo -e "-p PRODUCT\tSpecify the product, e.g. opensearch or opensearch-dashboards, make sure this is the name of your config folder and the name of your .tgz defined in dockerfile."
+    echo -e "-a ARCHITECTURE\tSpecify the multiple architecture you want to add to the multi-arch image, separate by comma, e.g. 'x64,arm64'."
```

Comment:
Please add checks to see if the user has entered valid args. eg: ar64 also starts running the script which should not

Reply:
It wont run to end because it will be disabled later on due to wget/docker command failure.

Reply:
Early failure is better than late failures. Else it becomes frustrating when you don't know what error you made

Reply:
It requires consistent editing later if we need to add more arch.
I really dont see this as a issue as it only complicated the script.
It is good to add if the script never has a safe guard, but I doubt it is really needed because in your logic even version input needs to be checked, product name as well, then repository validation, etc.

Reply:
I agree with Sayali. Fail fast.

Reply:
Add checks.